tests/integration/fixtures/csp/firefox_blocked_asset_output.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Wouldn't Firefox be sending more data than this?

Reply:
Yeah, if I understand these tests correctly the report Firefox sends is in `firefox_blocked_asset_input.json` and this is the result of POSTing that violation report to Sentry.

I was having trouble running integration tests locally, so I copied `chrome_output.json` and was hoping the integration tests would tell me what the output should look like.

Reply:
My mistake, I misread on my phone and thought this was the request payload.

Reply:
No problem! You had me worried for a sec. I appreciate you getting to PR so quickly.